src/containers/cooperation-details/add-course-modal-modal/AddCourseTemplateModal.tsx

```diff
@@ -175,11 +190,13 @@ const AddCourseTemplateModal: FC<AddCourseTemplateModalProps> = ({
           onLevelChange={onLevelChange}
           onSubjectChange={onSubjectChange}
           resetFilters={resetFilters}
+          user={user}
+          userLoading={userLoading}
         />
       )}
 
       <SimpleBar style={styles.cardsScroll(showFilters)}>
-        {loading ? (
+        {loading || userLoading ? (
```

Comment:
`loading` is already just `userLoading` ? 
You renamed it above using `:` so `loading` unnecessary

Reply:
loading and userLoading are different

Reply:
> loading and userLoading are different variables

I didn't see loading below from useAxios